baggers.js

```diff
@@ -5596,29 +5596,34 @@ var data = {
           url: 'http://jef.binomed.fr',
         },
         {
-          name: 'Web',
+          name: 'Prezs',
```

Comment:
Incorrect value. Please see the supported list on https://github.com/brownbaglunch/bblfr_data#websites or update the README :)